src/library/commissioner_impl.cpp

```diff
@@ -161,11 +161,11 @@ CommissionerImpl::CommissionerImpl(struct event_base *aEventBase)
     , mJoinerInfoRequester(nullptr)
     , mCommissioningHandler(nullptr)
 {
-    mBrClient.AddResource(mResourceUdpRx);
-    mBrClient.AddResource(mResourceRlyRx);
-    mProxyClient.AddResource(mResourceDatasetChanged);
-    mProxyClient.AddResource(mResourcePanIdConflict);
-    mProxyClient.AddResource(mResourceEnergyReport);
+    ASSERT(mBrClient.AddResource(mResourceUdpRx) == Error::kNone);
```

Comment:
Define another macro to simplify these codes?
```suggestion
    MustSuccess(mBrClient.AddResource(mResourceUdpRx));
```

Reply:
good idea. I have the same idea in my head, but leave it to future PR.